src/Microsoft.Health.Fhir.Liquid.Converter/Processors/FhirProcessor.cs

```diff
@@ -4,24 +4,25 @@
 // -------------------------------------------------------------------------------------------------
 
 using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Microsoft.Health.Logging.Telemetry;
 
 namespace Microsoft.Health.Fhir.Liquid.Converter.Processors
 {
     public class FhirProcessor : JsonProcessor
     {
-        public FhirProcessor(ProcessorSettings processorSettings)
-            : base(processorSettings)
+        public FhirProcessor(ProcessorSettings processorSettings, ITelemetryLogger telemetryLogger)
+            : base(processorSettings, telemetryLogger)
         {
         }
 
-        public override string Convert(string data, string rootTemplate, ITemplateProvider templateProvider, TraceInfo traceInfo = null)
+        protected override string InternalConvert(string data, string rootTemplate, ITemplateProvider templateProvider, TraceInfo traceInfo = null)
         {
             // TO DO For FHIR Specific Logic
             // 1. Add FHIR related checks for pre-processor and post-processor.
             // 2. Add log for version conversion.
             // 3. Add logic to process "extension" and "contained" fields.
 
-            return base.Convert(data, rootTemplate, templateProvider, traceInfo);
+            return base.InternalConvert(data, rootTemplate, templateProvider, traceInfo);
```

Comment:
Should be `base.InternalConvertFromObject` instead.

Reply:
Actually, this for now should be "base.Convert" defined in JsonProcessor. It's now just reusing logic of Json transformation.

Reply:
Oh totally missed the fact that it is inheriting from JsonProcessor and not BaseProcessor.